benchmarks/gluon_bench/tools/compare_bench_output.go

```diff
@@ -0,0 +1,119 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"time"
+)
+
+// we redefine the json statistic types here since we don't care about the extra information
+
+type JSONBenchmarkStatistics struct {
+	Total        time.Duration
+	Average      time.Duration
+	Fastest      time.Duration
+	Slowest      time.Duration
+	Median       time.Duration
+	Percentile90 time.Duration
+	Percentile10 time.Duration
+	RMS          time.Duration
+	SampleCount  int
+}
+
+type JSONBenchmarkReport struct {
+	Name       string
+	Runs       []*JSONBenchmarkStatistics
+	Statistics *JSONBenchmarkStatistics
+}
+
+func loadReportFromFile(path string) ([]*JSONBenchmarkReport, error) {
+	contents, err := ioutil.ReadFile(path)
```

Comment:
ioutil is deprecated, use `os.ReadFile` instead

Reply:
Done